server/index.js

```diff
@@ -0,0 +1,35 @@
+const express = require('express');
+const path = require('path');
+const db = require('./db/connection');
+const PORT = process.env.HOST_PORT || 8080;
+const app = express();
+const { seed } = require('../seed');
+const morgan = require('morgan');
+
+// for customizing env variables, .env needs to stay in root directory
+require('dotenv').config();
+
+// static middleware, body parsing middleware, logging middleware
+app.use(express.static(path.join(__dirname, '..', 'public')));
+app.use(express.json());
+app.use(morgan('dev'));
```

Comment:
Wont always be 'dev'.

Reply:
@naimoon6450 @dr3mit not sure if this is dumb but created a new env variable called `MORGAN_MODE` and in my.env have `MORGAN_MODE=dev`.

Then replaced `app.use(morgan('dev'))` with:
`app.use(morgan(process.env.MORGAN_MODE || null ));` because it has a default output of `process.stdout` if no mode is passed in, which I think is what we want in production.